### PR TITLE
perlPackages.RPM2: init at 1.4

### PR DESCRIPTION
**Pull request by invokesus**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -16086,6 +16086,21 @@ let
...
   };
 
+    RPM2 = buildPerlModule {
+    pname = "RPM2";
+    version = "1.4";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/L/LK/LKUNDRAK/RPM2-1.4.tar.gz";
+      sha256 = "5ecb42aa69324e6f4088abfae07313906e5aabf2f46f1204f3f1de59155bb636";
+    };
+    buildInputs = [ ModuleBuild pkgs.pkg-config pkgs.rpm ];
```
Comment (volth) [inline pkgs/top-level/perl-packages.nix:16096]:
```suggestion
    buildInputs = [ pkgs.pkg-config pkgs.rpm ];
```
`buildPerlModule` adds it

**Comment by invokesus** [comment]
Fixed. :)

**Review by stigtsp — APPROVED** [review]
Looks good to me

**Comment by aanderse** [comment]
@GrahamcOfBorg build perlPackages.RPM2 perldevelPackages.RPM2

**Comment by aanderse** [comment]
@invokesus please address build failure.

**Comment by invokesus** [comment]
@aanderse Added `meta.platforms` to restrict it to only Linux.

**Comment by stigtsp** [comment]
@GrahamcOfBorg build perlPackages.RPM2 perldevelPackages.RPM2

**Comment by stigtsp** [comment]
@invokesus Thx for the contribution.

**Pull request merged**